webui/react/src/components/WebhookCreateModal.tsx

```diff
@@ -13,33 +13,61 @@ import { routeToReactUrl } from 'utils/routes';
 
 const FORM_ID = 'create-webhook-form';
 
+const triggerEvents = [RunState.Completed, RunState.Error, V1TriggerType.TASKLOG] as const;
```

Comment:
<img width="412" alt="Screen Shot 2023-11-16 at 1 43 09 PM" src="https://github.com/determined-ai/determined/assets/643918/93cf5bf4-433e-4daa-902e-5968361b90cd">

Is there a way that the tasklog option can be read TASKLOG here to be more like the other types?